skins/oasis/modules/templates/PageHeader_Index.php

```diff
@@ -25,17 +25,12 @@
 				<? endif; ?>
 			</div>
 			<div class="header-column header-tally">
-				<?php if ( !empty( $pageExists ) ) { ?>
-					<div id="PageShareContainer" class="page-share-container">
-						<?= F::app()->renderView( 'PageShare', 'Index' ); ?>
-					</div>
-				<?php } ?>
 				<? if ( !is_null( $tallyMsg ) ): ?>
 					<div class="tally"><?= $tallyMsg ?></div>
+					<a href="/wiki/Special:CreatePage?flow=create-page-contribute-button" class="wikia-button createpage" data-id="createpage">
```

Comment:
Hardcoded url was meant to be a temporary solution for ABTest. For production ready solution please use `SpecialPage::getTitleFor( $specialPageName )->getLocalURL()`.

Bonus points for reusing existing logic in: https://github.com/Wikia/app/blob/55a5452c79a44f0b49e6886f0352fba87745ecaf/skins/oasis/modules/ContributeMenuController.class.php#L19-L19 which already includes flow tracking parameters.